docusaurus/docs/dev-docs/api/plugins/admin-panel-api.md

```diff
@@ -26,17 +26,17 @@ The entry file for the Admin Panel API is `[plugin-name]/admin/src/index.js`. Th
 | Lifecycle functions | <ul><li> [register](#register)</li><li>[bootstrap](#bootstrap)</li></ul> |
 | Async function      | [registerTrads](#registertrads)                                          |
 
-## Lifecycle functions
+## Available methods
```

Comment:
@pwizla 

I think `methods` is better here than `functions`. Functions is the most generic term. I think I would prefer `Available callbacks` even more. What do you think?

Reply:
I love the idea! Let's go for this one ("callbacks"). Thank you for your precious, constructive feedback, Gustav! 🤗